articles/architecture-scenarios/_includes/_keep-reading.md

```diff
@@ -0,0 +1,78 @@
+<% if (platform === "b2b") { %>
+<!-- Exclude current page from keep reading section -->
+<% if (self !== "architecture") { %>
+* [Architecture](/architecture-scenarios/implementation/b2b/b2b-architecture)
```

Comment:
```suggestion
* [Architecture](/architecture-scenarios/implementation/${platform}/${platform}-architecture)
```

Reply:
Does `${platform}` interpolation syntax work for lodash template? I thought it only supported limited ejs syntax i.e. `<%=platform%>`.

Reply:
Did a quick "isolated test" and `${variable}` **does** work in lodash... 
Let me try in docs and update the PR.

Reply:
@mostekcm please review the last commit and approve....